tensorflow_lite_support/c/task/core/utils/base_options_utils.cc

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow_lite_support/c/task/core/utils/base_options_utils.h"
+
+namespace tflite {
+namespace task {
+
+TfLiteBaseOptions GetDefaultBaseOptions() {
+  TfLiteBaseOptions base_options = {0};
+  base_options.compute_settings.cpu_settings.num_threads = -1;
+  return base_options;
+}
+
+}  // namespace task
+}  // namespace tflite
```

Comment:
nit: add an end-of-file blank line. Same for the other files (where you see the red sign of a circle with a `-`.).